### PR TITLE
feat: SimBrief integration command

### PR DESCRIPTION
**Pull request by nebula231**
feat: SimBrief integration command

The command shows users how to use SimBrief integration, with an image showing how it will look when everything is done correctly, and to the docs referring to how to set it up.  Commands are: import, integration,  integ

image

zx#0671

### CODE REVIEW

File: src/commands/a32nx/simbriefimport.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const Simbrief_request_URL = 'https://cdn.discordapp.com/attachments/945012754589298721/945056041656279130/unknown.png';
+
+export const simbriefimport: CommandDefinition = {
+    name: ['import', 'integration', 'integ'],
+    description: 'Shows how to use SimBrief integration ',
```
Comment (benw202) [inline src/commands/a32nx/simbriefimport.ts:9]:
```suggestion
    description: 'Shows how to use SimBrief integration',
```

Removes space

**Review by benw202 — CHANGES_REQUESTED** [review]
Good first command!

**Review by benw202 — APPROVED** [review]

**Comment by benw202** [comment]
LGTM!

**Review by NUTTTTTTTT — APPROVED** [review]
LGTM! 
image

**Pull request merged**